src/Microsoft.Identity.Web/Hosts/PlainDotNet/PlainDotNetTokenAcquisitionHost.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System.Net;
+using System.Security.Claims;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Options;
+using Microsoft.Identity.Client;
+using Microsoft.IdentityModel.Tokens;
+
+namespace Microsoft.Identity.Web.Hosts
+{
+    internal class PlainDotNetTokenAcquisitionHost : ITokenAcquisitionHost
+    {
+
+        IOptionsMonitor<MicrosoftIdentityOptions> _optionsMonitor;
+        IOptionsMonitor<MergedOptions> _mergedOptionsMonitor;
+        IOptionsMonitor<ConfidentialClientApplicationOptions> _ccaOptionsMonitor;
+
+        public PlainDotNetTokenAcquisitionHost(
+            IOptionsMonitor<MicrosoftIdentityOptions> optionsMonitor, 
+            IOptionsMonitor<MergedOptions> mergedOptionsMonitor,
+            IOptionsMonitor<ConfidentialClientApplicationOptions> ccaOptionsMonitor)
+        {
+            _optionsMonitor = optionsMonitor;
+            _mergedOptionsMonitor = mergedOptionsMonitor;
+            _ccaOptionsMonitor = ccaOptionsMonitor;
+        }
+
+        Task<ClaimsPrincipal?> ITokenAcquisitionHost.GetAuthenticatedUserAsync(ClaimsPrincipal? user)
```

Comment:
What is the reason for using explicit interface implementation? Is it to be able to keep this class internal? Needs a comment ..